projects/core/components/dialog/dialog.service.ts

```diff
@@ -1,24 +1,29 @@
-import {Injectable} from '@angular/core';
-import {AbstractTuiDialogService} from '@taiga-ui/cdk';
+import {Inject, inject, Injectable} from '@angular/core';
+import {AbstractTuiDialogService, TuiIdService} from '@taiga-ui/cdk';
 import {TuiDialogOptions} from '@taiga-ui/core/interfaces';
+import {TUI_DIALOG_OPTIONS} from '@taiga-ui/core/tokens/dialog-options';
 import {PolymorpheusComponent} from '@tinkoff/ng-polymorpheus';
 
 import {TuiDialogComponent} from './dialog.component';
 
 const DIALOG = new PolymorpheusComponent(TuiDialogComponent);
-const DEFAULT_OPTIONS = {
-    size: `m`,
-    required: false,
-    closeable: true,
-    dismissible: true,
-    label: ``,
-    header: ``,
-} as const;
 
 @Injectable({
     providedIn: `root`,
 })
 export class TuiDialogService extends AbstractTuiDialogService<TuiDialogOptions<any>> {
     protected readonly component = DIALOG;
-    protected readonly defaultOptions: TuiDialogOptions<any> = DEFAULT_OPTIONS as any;
+    protected readonly defaultOptions: TuiDialogOptions<any> = {
+        ...inject(TUI_DIALOG_OPTIONS),
+        data: undefined,
+    };
+
```

Comment:
@splincode so, since having a token for each dialog is non-negotiable, the only thing that could help a little is a helper function that would look something like this: 

![image](https://user-images.githubusercontent.com/37042257/188730411-0137247d-a960-4136-a413-feef53ca2891.png)

but even this does not help too much, as doing this manually is rather simple and trivial anyways. So, what do you think? Should I use this function, or just go the simple way?

Reply:
just go the simple way))